ext/standard/net.c

```diff
@@ -177,7 +177,11 @@ PHP_FUNCTION(net_get_interfaces) {
 	dwRetVal = GetAdaptersAddresses(family, flags, NULL, pAddresses, &outBufLen);
 
 	if (NO_ERROR != dwRetVal) {
-		/* TODO check GetLastError() */
+		char *buf = php_win32_error_to_msg(GetLastError());
+		if (buf) {
```

Comment:
[`php_win32_error_to_msg()`](https://github.com/php/php-src/blob/5de750f41c1ee3ed834ebf4f811ba03a6cc2d9bc/win32/winutil.c#L28-L47) never returns NULL, so no need to check for this condition.

Reply:
true...